src/libraries/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -81,7 +81,15 @@ public void CustomAttributes()
         [Fact]
         public void FullyQualifiedName()
         {
-            Assert.Equal(Assembly.GetExecutingAssembly().Location, Module.FullyQualifiedName);
+            var loc = AssemblyPathHelper.GetAssemblyLocation(Assembly.GetExecutingAssembly());
+
+            // Browser will include the path (/), so strip it
```

Comment:
hm so what does `Module.FullyQualifiedName` return for an in-memory assembly, does it also return an empty string like `Assembly.Location` did? it sounds like maybe we should fix/follow-up on this.